Bookings

zdevaty opened the pull request.

Commit:
Add OpenAPI description of endpoints for bookings

Commit:
Add Portier API description

Commit:
Remove redundant file

Commit:
Generate API server for bookings

Commit:
Add create and list bookings

andreas-goebel commented on apiservices/api_booking_service.go:
```diff
@@ -18,45 +18,166 @@ package apiservices
+
+	errChan := make(chan error)
+	go func() {
+		err := graph.TestUserRequest(context.Background())
```
Is this a "test" request or why is this request named "test..."?

Commit:
Implement deleting bookings

Commit:
Improve method name

Not to be confused with unit tests

Commit:
Implement sending notifications about arriving guests

This implementation contains finding the event among some rondom user.
That does not work as it is, but I want to preserve it for reference.

Commit:
Make recipient a parameter
